README.md

```diff
@@ -11,16 +11,23 @@ Chrome Platform Status
 
 ### Installation
 1. Before you begin, make sure that you have a java JRE (version 8 or greater) installed. JRE is required to use the DataStore Emulator.
-1. Install global CLIs
+1. Install global CLIs in the home directory
     1. [Google App Engine SDK for Python](https://cloud.google.com/appengine/docs/standard/python3/setting-up-environment). Make sure to select Python 3.
     1. pip, node, npm.
     1. Gulp `npm install --global gulp-cli`
-1. Install npm dependencies `npm ci`
-1. Create a virtual environment.
+1. We recommend using an older node version, e.g. node 10
+    1. Use `node -v` to check the default node version
+    2. `nvm use 10` to switch to node 10
```

Comment:
I have node 12 installed and it seems to work.  If we could bump this up to 12, that might be better somehow.